app/src/main/java/com/android/voyageur/ui/trip/activities/Activities.kt

```diff
@@ -32,9 +74,204 @@ fun ActivitiesScreen(
             selectedItem = navigationActions.currentRoute())
       },
       content = { pd ->
-        Text(
-            modifier = Modifier.padding(pd).testTag("emptyActivitiesPrompt"),
-            text =
-                "You're viewing the Activities screen for ${trip.name}, but it's not implemented yet.")
+        LazyColumn(
+            modifier =
+                Modifier.padding(pd).padding(top = 16.dp).fillMaxWidth().testTag("lazyColumn"),
+            verticalArrangement = Arrangement.spacedBy(10.dp, Alignment.Top),
+        ) {
+          item {
+            Text(
+                text = "Drafts",
+                fontSize = 24.sp,
+                fontWeight = FontWeight.Bold,
+                modifier = Modifier.padding(start = 10.dp))
+          }
+          drafts.forEach { activity ->
+            item {
+              ActivityItem(activity)
+              Spacer(modifier = Modifier.height(10.dp))
+            }
+          }
+          item {
+            Text(
+                text = "Final",
+                fontSize = 24.sp,
+                fontWeight = FontWeight.Bold,
+                modifier = Modifier.padding(start = 10.dp))
+          }
+          final.forEach { activity ->
+            item {
+              ActivityItem(activity)
+              Spacer(modifier = Modifier.height(10.dp))
+            }
+          }
+        }
+      })
+}
+
+/** Composable that displays an activity item in a card view. */
+@Composable
+fun ActivityItem(
+    activity: Activity,
+) {
+  var isExpanded by remember { mutableStateOf(false) }
+
+  val dateFormat = SimpleDateFormat("dd/MM/yyyy", Locale.getDefault())
+  val timeFormat = SimpleDateFormat("hh:mm a", Locale.getDefault())
```

Comment:
It might be better to use DateTimeFormatter, such that you take into account user's settings

Reply:
Thanks for the suggestion! Changed to use DateTimeFormatter